core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/labelingrules/NameConverter.java

```diff
@@ -30,34 +30,25 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
  * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.openjdk.jmc.ui.common.labelingrules;
+package org.openjdk.jmc.common.labelingrules;
 
 import java.text.MessageFormat;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.logging.Level;
 
-import org.eclipse.core.runtime.IConfigurationElement;
-import org.eclipse.core.runtime.IExtension;
-import org.eclipse.core.runtime.IExtensionPoint;
-import org.eclipse.core.runtime.IExtensionRegistry;
-import org.eclipse.core.runtime.Platform;
 import org.openjdk.jmc.common.jvm.JVMCommandLineToolkit;
 import org.openjdk.jmc.common.jvm.JVMDescriptor;
+import org.openjdk.jmc.common.messages.internal.Messages;
+import org.openjdk.jmc.common.resource.Resource;
 import org.openjdk.jmc.common.util.Environment;
-import org.openjdk.jmc.ui.common.CorePlugin;
-import org.openjdk.jmc.ui.common.idesupport.IDESupportFactory;
-import org.openjdk.jmc.ui.common.resource.Resource;
 
 /**
  * Converts names according to rules specified with the {@value #LABELING_RULES_EXTENSION_POINT}
```

Comment:
This causes a Javadoc error, same in line 143 ` * {@value #LABELING_RULES_EXTENSION_POINT} extension point.`